src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/utils/RespondentBuilder.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.utils;
+
+import uk.gov.hmcts.et.common.model.ccd.Address;
+import uk.gov.hmcts.et.common.model.ccd.types.RespondentSumType;
+
+import static uk.gov.hmcts.ecm.common.model.helper.Constants.YES;
+
+public class RespondentBuilder {
```

Comment:
Nice class, but we should have JavaDoc around it

Reply:
Cant we just create a class with fields and make a lomboks @Builder over it?

Reply:
This is what I love about code reviews.  I never knew about the lambok @builder until you've mentioned it Ivan.  Nice suggestion :)
Jack, not sure if it's possible within the way you've built this, but certainly a great suggestion to make life easier in future.

Reply:
Lombok builder looks really cool, correct me if I'm wrong but from what I've read it looks like the @Builder would go on the respondent object in data-model? We use @Data for this right now which does give us auto-generated get/setters (also really cool, saves a ton of time) but the advantage of using this builder here is that we can rollup multiple related properties into a single setter (for example `withExtension` takes in three related properties and sets them). I don't think @Builder is applicable in this scenario, but its definately something to keep in mind in future

Reply:
With builder you can build these 3 or 4 properties according to the situation and return this object and then add more fields and built it only in the final stage(absolute free to use with different situations), but maybe in this case it is more informative when you are marking what have you done within method.